O/openfhe_julia/common.jl

```diff
@@ -107,7 +107,7 @@ function prepare_openfhe_julia_build(name::String, git_hash::String)
     # Dependencies that must be installed before this package can be built
     dependencies = [
         BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.9")),
```

Comment:
Not strictly necessary but better to do this, too:
```suggestion
        BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.11.0")),
```